### PR TITLE
💊 Optimize Python >=3.8 type comments support method i90.

### PR DESCRIPTION
**Pull request by hadialqattan**

### CODE REVIEW

**Review by isidentical — COMMENTED** [review]
Instead of trying version check on every possible node, you can just create a function named `def _type_comment(node: ast.AST) -> None` which does the job by using `getattr`, not the version check, since we might switch to `typed_ast` to use this feature on all possible versions. Example;
```
def type_comment(node: ast.AST) -> None:
      if isinstance(node, ASTFunctionT): mode="func_type"
      else: mode="eval"
      type_comment = getattr(node, "type_comment", None)
      if type_comment is not None: ....
```

**Comment by hadialqattan** [comment]
> Instead of trying version check on every possible node, you can just create a function named `def _type_comment(node: ast.AST) -> None` which does the job by using `getattr`, not the version check, since we might switch to `typed_ast` to use this feature on all possible versions. Example;
> 
> ```
> def type_comment(node: ast.AST) -> None:
>       if isinstance(node, ASTFunctionT): mode="func_type"
>       else: mode="eval"
>       type_comment = getattr(node, "type_comment", None)
>       if type_comment is not None: ....
> ```

I agree with you!

**Comment by isidentical** [comment]
By the way, do you like to join our telegram group?

File: unimport/scan.py [hunk 1]
```diff
@@ -237,6 +234,16 @@ def scan(self, source: str) -> None:
         self.names = list(self.get_names())
         self.unused_imports = list(self.get_unused_imports())
 
+    def _type_comment(self, node: ast.AST) -> None:
+        ASTFunctionT = (ast.FunctionDef, ast.AsyncFunctionDef)
```
Comment (isidentical) [inline unimport/scan.py:238]: Declare this is a global

Comment (isidentical) [reply unimport/scan.py:238]: Also use it on L111

Comment (hadialqattan) [reply unimport/scan.py]: Good idea!

**Comment by hadialqattan** [comment]
> By the way, do you like to join our telegram group?

Of course!

**Comment by hadialqattan** [comment]
> > By the way, do you like to join our telegram group?
> 
> Of course!

I hope it's an English group because I can't speak Turkish!

**Comment by isidentical** [comment]
> I hope it's an English group because I can't speak Turkish!

Indeed!

**Comment by hadialqattan** [comment]
> > I hope it's an English group because I can't speak Turkish!
> 
> Indeed!

Ok, so please send the invitation link to <alqattanhadizaki@gmail.com>

**Comment by hakancelik96** [comment]
> > > I hope it's an English group because I can't speak Turkish!
> > 
> > 
> > Indeed!
> 
> Ok, so please send the invitation link to [alqattanhadizaki@gmail.com](mailto:alqattanhadizaki@gmail.com)

You are in the group, check the telegram.

**Comment by hadialqattan** [comment]
> > > > I hope it's an English group because I can't speak Turkish!
> > > 
> > > 
> > > Indeed!
> > 
> > 
> > Ok, so please send the invitation link to [alqattanhadizaki@gmail.com](mailto:alqattanhadizaki@gmail.com)
> 
> You are in the group, check the telegram.

Thanks a lot!

**Review by isidentical — APPROVED** [review]

**Pull request merged**